proto/dydxprotocol/vault/query.proto

```diff
@@ -25,10 +25,13 @@ service Query {
   rpc AllVaults(QueryAllVaultsRequest) returns (QueryAllVaultsResponse) {
     option (google.api.http).get = "/dydxprotocol/vault/vault";
   }
-  // Queries owner shares of a vault.
+  // Queries total shares.
+  rpc TotalShares(QueryTotalSharesRequest) returns (QueryTotalSharesResponse) {
+    option (google.api.http).get = "/dydxprotocol/vault/total_shares";
```

Comment:
nit: Should the path include a reference to `megavault` rather than just `vault`?

Reply:
good idea